cwltool/process.py

```diff
@@ -205,15 +205,26 @@ def adjustFilesWithSecondary(rec, op, primary=None):
             adjustFilesWithSecondary(d, op, primary)
 
 
-def stageFiles(pm, stageFunc, ignoreWritable=False):
-    # type: (PathMapper, Callable[..., Any], bool) -> None
+def stageFiles(pm, stageFunc=None, ignoreWritable=False, symFunc=True):
+    # type: (PathMapper, Callable[..., Any], bool, bool) -> None
```

Comment:
`symFunc` is type `bool`  but the name implies that it is a function. Would `symlink` be a better name?